src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1522,6 +1522,22 @@ static bool processSubscriptions
                                                  tenant,
                                                  xauthToken))
     {
+      long long rightNow = getCurrentTime();
+
+      //
+      // If broker running without subscription cache, put lastNotificationTime and count in DB 
+      //
+      if (mongoSubCacheActive == false)
+      {
+        BSONObj query  = BSON("_id" << OID(mapSubId));
+        BSONObj update = BSON("$set" <<
+                              BSON(CSUB_LASTNOTIFICATION << rightNow) <<
+                              "$inc" << BSON(CSUB_COUNT << 1));
+        
+        collectionUpdate(getSubscribeContextCollectionName(tenant), query, update, false, err);
```

Comment:
collectionUpdate has its own error reporting, I didn't feel the need to add an extra error here.

There is not much to do if this fails ...

The only thing I could possible do is to not update the cache if collectionUpdate fails.
But ... why "destroy" the cache just because collectionUpdate failed?